addons/mail/static/src/components/composer/composer.js

```diff
@@ -208,10 +208,15 @@ export class Composer extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickCaptureGlobal(ev) {
+    async _onClickCaptureGlobal(ev) {
         if (this.contains(ev.target)) {
             return;
         }
+        // Let event be handled by bubbling handlers first
+        await new Promise(this.env.browser.setTimeout);
+        if (isEventHandled(ev, 'MessageActionList.replyTo')) {
+            return;
+        }
```

Comment:
I suggested earlier to move the "reply" handle here instead, wasn't that possible?

But I guess we're out of time now, if that works whatever.

Reply:
I'm not sure how that would be possible? I think your suggestion was to move the reply method to the message model, the logic on whether to discard the current message if it's already opened to the discuss model, which I did. I'm confused about this comment.